README.md

```diff
@@ -105,7 +105,7 @@ module "example" {
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
 | ami\_owner\_account\_id | The ID of the AWS account that owns the OpenVPN AMI, or "self" if the AMI is owned by the same account as the provisioner. | `string` | `"self"` | no |
-| aws\_instance\_type | The AWS instance type to deploy (e.g. t3.medium). | `string` | `"t3.small"` | no |
+| aws\_instance\_type | The AWS instance type to deploy (e.g. t4g.medium). | `string` | `"t4g.small"` | no |
```

Comment:
Shouldn't we just make our example text match the default value here?
```suggestion
| aws\_instance\_type | The AWS instance type to deploy (e.g. t4g.small). | `string` | `"t4g.small"` | no |
```

Reply:
Oh for Pete's sake!  Please see commit 66f8478.

Reply:
Sorry- I only said something because it looks odd since we do that nearly everywhere else.  Thanks for changing it.